src/util.rs

```diff
@@ -306,18 +306,18 @@ pub fn hex_or_bool(n: u32, width: u32) -> Tokens {
     }
 }
 
-/// Turns `n` into an unsuffixed literal
-pub fn unsuffixed(n: u64) -> Lit {
-    Lit::Int(n, IntTy::Unsuffixed)
+/// Turns `n` into an unsuffixed token
+pub fn unsuffixed(n: u64) -> Tokens {
+    let mut t = Tokens::new();
+    t.append (format!("{}", n));
```

Comment:
nit: stray whitespace between `append` and `(`

Reply:
You'll find a lot more nits if you run rustfmt over it, which is preventing me from doing it all the time. 😉